### PR TITLE
ft: ZENKO-1389: md proxy parallel route

### PR DESCRIPTION
**Pull request by ploki**

### CODE REVIEW

**Comment by ploki** [comment]
@bert-e reset

File: lib/storage/metadata/proxy/BucketdRoutes.js [hunk 1]
```diff
@@ -224,7 +232,7 @@ class BucketdRoutes {
      */
     _objectRoutes(req, res, uriComponents, logger) {
         if (uriComponents.bucketName === undefined) {
-            logger.error('Missing bucket name for bucket route',
+            logger.error('Missing bucket name for object route',
```
Comment (bennettbuchanan) [inline lib/storage/metadata/proxy/BucketdRoutes.js:235]: 👍

File: lib/storage/metadata/proxy/BucketdRoutes.js [hunk 2]
```diff
@@ -254,6 +262,33 @@ class BucketdRoutes {
         }
     }
 
+    /**
+     * Handle routes related to operations on both objects and buckets
+     *
+     * @param {http.IncomingMessage} req - request being processed
+     * @param {http.OutgoingMessage} res - response associated to the request
+     * @param {object} uriComponents -
```
Comment (bennettbuchanan) [inline lib/storage/metadata/proxy/BucketdRoutes.js]: Minor: can you add some quick descriptions of these params?

File: tests/functional/metadataProxy/routesToMem.js [hunk 3]
```diff
@@ -34,13 +34,13 @@ const bucketInfo = {
     deleted: false,
     serverSideEncryption: null,
     versioningConfiguration: null,
-    websiteConfiguration: null,
```
Comment (bennettbuchanan) [inline tests/functional/metadataProxy/routesToMem.js:37]: 💭I don't follow why we need to remove this.

**Review by bennettbuchanan — APPROVED** [review]
Overall LGTM. I've left some thoughts.

File: tests/functional/metadataProxy/routesToMem.js (hunk 3 above)
Comment (ploki) [reply tests/functional/metadataProxy/routesToMem.js:37]: It appears to not be served back to the client

**Comment by ploki** [comment]
@bert-e approve

**Review by rachedbenmustapha — APPROVED** [review]

**Comment by ploki** [comment]
@bert-e reset

**Pull request merged**